content/applications/general/email_communication/faq.rst

```diff
@@ -1,173 +1,225 @@
-  examples of questions whose answers can be useful:
+   The two most recent days (today and yesterday) are not compressed, while the older ones are, in
+   order to save space. The naming of the files for today and yesterday are respectively:
+   :file:`odoo.log` and :file:`odoo.log.1`.
 
-  - Is this simply a reply from an email going out from Odoo ?
+   For the following days, they are named with their dates, and then compressed. Use the command
+   :command:`grep` and :command:`zgrep` (for the compressed ones) to search through the files.
 
-  - Are you using an incoming email server or somehow redirecting?
+   .. seealso::
+      For more information on logs and how to access them via the Odoo.sh dashboard see :ref:`this
+      administration documentation <odoosh/logs>`.
 
-  - Can you provide us with an example of an email that has been correctly forwarded ?
+      For more information on accessing logs via the command line visit :ref:`this developer
+      documentation <reference/cmdline/server/logging>`.
 
-- Providing answers to the following questions:
+- **Odoo Online** users won't have access to the logs. However `Odoo Support
+  <https://www.odoo.com/help>`_ can be contacted if there is a recurring issue with the same client
+  or domain.
 
-  - Is it a generic issue or is it specific to a use case? If yes, which one exactly?
+Get help from Odoo support
+==========================
 
-  - Is it working as expected? In case the email is sent using Odoo, the bounce email should reach
-    the Odoo database and display the :ref:`red envelope <red_envelop>`.
+In order to get helped efficiently, please provide as much information as possible. Here is a list
+of what can be helpful when reaching out to the Odoo Support team about an issue:
+
+#. Send a copy of the email headers. The `.EML` file (or **headers**) of the email is the file
+   format containing all the technical information required for an investigation. The documentation
+   from the email provider might explain the process on how to get EML file/header files. Once the
+   headers of the email are obtained, adding it into the Odoo Support ticket is the most efficient
+   way for the Odoo Support team to investigate.
+
+   .. seealso::
+      - `Gmail documentation on headers
+        <https://support.google.com/mail/answer/29436>`_
+      - `Outlook documentation on headers
+        <https://support.microsoft.com/en-us/office/view-internet-message-headers-in-outlook-cd039382-dc6e-4264-ac74-c048563d212c#tab=Web>`_
+
+#. Explain the exact flow that is being followed to normally receive those emails in Odoo. Here are
+   examples of questions whose answers can be useful:
+
+   - Is this simply a reply from an email going out from Odoo?
+   - Is there an incoming email server being used, or is the email somehow being redirected?
+   - Is there an example of an email that has been correctly forwarded?
+
+#. Provide answers to the following questions:
```

Comment:
For some reason in the HTML version, it looks like there should be another (blank) paragraph between this line and the previous line

Reply:
Adding this in doesn't make a difference, the lines are skipped.